server.go

```diff
@@ -0,0 +1,79 @@
+package sqrl
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"sync"
+)
+
+var mutex = &sync.Mutex{}
+
+func com(w http.ResponseWriter, req *http.Request) {
+	mutex.Lock()
+	c, err := GetCPUInfo()
+	if err != nil {
+		log.Fatal(err)
+	}
+	mutex.Unlock()
+
+	c.LogWriter()
+
+	fmt.Println(c)
+}
+
+func ros(w http.ResponseWriter, req *http.Request) {
+	mutex.Lock()
+	r, o, s, err := RamOSInfo()
+	if err != nil {
+		log.Fatal(err)
+	}
+	mutex.Unlock()
+
+	r.LogWriter()
+	o.LogWriter()
+	s.LogWriter()
+
+	fmt.Println(r)
```

Comment:
Instead of printing here, make use of the verbose flag. If you check out main.go, you'll remember that we check if the verbose flag is set and then we print out.  Fix that please.